teamsite/main/static/main/js/script.js

```diff
@@ -0,0 +1,18 @@
+const header = document.querySelector("header");
+
+window.addEventListener ("scroll", function() {
+	header.classList.toggle ("sticky", window.scrollY > 0);
+});
+
+let menu =  document.querySelector('#menu-icon');
```

Comment:
Why do we have these variables in global scope of file? Maybe it's better to create some entry-point like func, and call it any time we want to load the script?